README.md

```diff
@@ -159,6 +159,57 @@ echo topsecretpassword > ~/.config/openstack-password.txt
 ./pkb.py --cloud=OpenStack --benchmarks=ping
 ```
 
+## Kubernetes configuration and credentials
+Perfkit uses `kubectl` binary in order to communicate with Kubernetes cluster - you need to pass the path to `kubectl` binary using `--kubectl` flag.  
+Authentication to Kubernetes cluster is done via `kubeconfig` file. Its path is passed using `--kubeconfig` flag.
```

Comment:
Nitpick (not necessary for merge) - could you link to the kubeconfig documentation here? https://github.com/kubernetes/kubernetes/blob/release-1.0/docs/user-guide/kubeconfig-file.md